tests/test_ipm.py

```diff
@@ -116,6 +116,21 @@ def test_ipm_motion(fake_ipm):
     # Remove IPM Diode
     ipm.diode.remove()
     assert ipm.diode.state.state.get() == 2
+    # Target_in
+    status = ipm.target_in(4)
+    assert ipm.target.state.get() == 4
+    status.wait(timeout=1)
```

Comment:
technically we should wait before checking the result, not after. This could matter if the fake device implementation changes later

```suggestion
    status.wait(timeout=1)
    assert ipm.target.state.get() == 4
```

Reply:
sounds good, thank you - fixed this in 371615e